examples/src/crypto_materials_manager/explicit_default.py

```diff
@@ -0,0 +1,76 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""
+The default cryptographic materials manager (CMM)
+is a CMM that uses a keyring or master key provider
+to manage the data key materials.
+When using the encrypt and decrypt APIs, as well as the caching CMM,
+you can supply either a CMM, a keyring, or a master key provider.
+If you supply a keyring or master key provider
+those components will automatically wrap it in a default CMM.
+
+This example shows how you could do this manually if you wanted to.
```

Comment:
This example I'm a little unsure is necessary. It is useful in terms of educating the user about the default CMM, but practically, without any additional functionality, it is pointless. Would it be enough to include the description of the default CMM in one of the other examples?

Reply:
Maybe. What I was going for is demonstrating how to use the default CMM if you need it, such as if you're making a custom CMM that you configure using keyrings. More explanation in the other examples might be sufficient, yeah.